filters/include/pcl/filters/passthrough.h

```diff
@@ -332,7 +298,7 @@ namespace pcl
       inline void
       getFilterLimitsNegative (bool &limit_negative) const
```

Comment:
This one does not have a corresponding method in `FilterIndices`, but I'd also deprecate this and advise users to utilize `use setNegative() with return value instead`.